server/statistics/hot_peer.go

```diff
@@ -27,27 +27,27 @@ const (
 
 // HotPeerStat records each hot peer's statistics
 type HotPeerStat struct {
-	StoreID  uint64 `json:"store_id"`
-	RegionID uint64 `json:"region_id"`
+	StoreID  uint64 `json:"store-id"`
```

Comment:
Seem store id is not necessary to output

Reply:
yep